source/2_platform/rex_windows/src/win_window.cpp

```diff
@@ -35,8 +35,11 @@ namespace rex
     //-------------------------------------------------------------------------
     Window::Window()
         : m_wnd_class()
+        , m_event_handler(this)
         , m_hwnd(nullptr)
-        , m_destroyed(false)
+        , m_min_width(200)
```

Comment:
note: hardcoded values here
will make issue for it

Reply:
Well you can but is this really required I mean do you wanna make the user adjust the min width and height?

Reply:
if you don't want the user to change this, then please put a comment somewhere explaining why it's 200

Reply:
I don't really have a reason for it to be 200, I think this is a small enough size to see something on the screen but if you have another opinion I'm all ears.

Reply:
200 is fine, I just like to see a reason somewhere why it's 200, this can be as simple as a comment dictating what you wrote above